### PR TITLE
Uni-merge WIDGET FIX

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

fixes widget setup, co-created with @alfetopito @nenadV91

### CODE REVIEW

File: src/custom/lib/components/Widget.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import { Provider as EthersProvider } from '@ethersproject/abstract-provider'
```
Comment (alfetopito) [inline src/custom/lib/components/Widget.tsx:1]: Can you rename this file to WidgetMod and create the index file and so on?
Will be easier when we have to update it :)

File: src/custom/lib/components/Widget.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  return (
+    <StrictMode>
+      <I18nProvider locale={locale}>
+        <ThemeProvider theme={theme}>
+          <WidgetWrapper width={width} className={className}>
+            <DialogWrapper ref={setDialog} />
+            <DialogProvider value={userDialog || dialog}>
+              <ErrorBoundary onError={onError}>
+                <WidgetPropValidator {...props}>
+                  {/* MOD (128) */}
```
Comment (alfetopito) [inline src/custom/lib/components/Widget.tsx:128]: What does `128` mean?

**Review by alfetopito — APPROVED** [review]
Working good, only a comment regarding modding naming convention

**Comment by elena-zh** [comment]
I'm not sure if it is related to the current PR or not, but when I press on the 'Buy COW' button, I see that 'Select token' in the From field is missing: 
image
image

**Comment by elena-zh** [comment]
@alfetopito , thank you! Will post it in the #374 PR.

File: src/custom/lib/components/Widget.tsx (hunk 2 above)
Comment (W3stside) [reply src/custom/lib/components/Widget.tsx:128]: line 128

Comment (W3stside) [reply src/custom/lib/components/Widget.tsx:128]: but doesn't make sense here does it. DOES IT!?? ill remove it

Comment (alfetopito) [reply src/custom/lib/components/Widget.tsx:128]: Oh 🤦 

Yeah, for dumb people like me it makes it confusing

**Pull request merged**